workspace_info.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
Update year to 2021

Reply:
Changing, though this may cause a failure in boilerplate validation.